src/popup/router/routes.js

```diff
@@ -508,7 +508,11 @@ export default [
     path: '*',
     component: NotFound,
     meta: {
-      ifNotAuth: true,
+      asModal: true,
+      backButton: true,
+      title: 'not-found',
+      closeButton: true,
+      hideNotificationsIcon: true,
```

Comment:
I suggest grouping the keys 
```suggestion
      title: 'not-found',
      asModal: true,
      backButton: true,
      closeButton: true,
      hideNotificationsIcon: true,
```